zerver/lib/push_notifications.py

```diff
@@ -615,6 +621,21 @@ def send_notifications_to_bouncer(
         increment=total_android_devices + total_apple_devices,
     )
 
+    deleted_android_tokens_list = cast(List[int], response_data["deleted_android_tokens"])
+    deleted_apple_tokens_list = cast(List[int], response_data["deleted_apple_tokens"])
```

Comment:
These should potentially assert like the code above; or maybe we should do something with pydantic to actually parse the expected format (being careful to not fail if there are extra fields). @andersk FYI as this may be best done as a follow-up by you.